src/hnsw/build.c

```diff
@@ -452,6 +463,14 @@ static void BuildIndex(
             // Unlock and release buffer
             UnlockReleaseBuffer(buffer);
         }
+        double             M = ldb_HnswGetM(index);
+        double             mL = 1 / log(M);
+        usearch_metadata_t meta = usearch_metadata(buildstate->usearch_index, &error);
```

Comment:
Is this the exact same code as above?
Can we factor it out into a `Check__` function?